### ISSUE TITLE
[Bug]: Error: No electron app entry file found

### ISSUE THREAD

**Issue by rappasoft** [NativePHP/electron#12]
### What happened?

Myself and a few others are getting all the way to the `native:serve` step and being presented with the following error:

```
 Would you like to start the NativePHP development server (yes/no) [no]:
 > yes

Starting NativePHP dev server…
Fetching latest dependencies…
Starting NativePHP app…
yarn run v1.22.19
warning package.json: No license field
$ electron-vite dev --watch
vite v4.3.9 building for development...

watching for file changes...

build started...
✓ 2 modules transformed.

build the electron main process successfully

-----

Could not resolve "../../resources/cacert.pem?asset&asarUnpack" from "src/main/index.js"
file: /Users/Rappa/Desktop/NativePHP/vendor/nativephp/electron/resources/js/src/main/index.js
vite v4.3.9 building for development...

watching for file changes...

build started...
transforming (1) src/preload/index.js"contextBridge" and "ipcRenderer" are imported from external module "electron" but never used in "src/preload/index.js".
✓ 1 modules transformed.
out/preload/index.js  0.70 kB
built in 99ms.

build the electron preload files successfully

-----

dev server running for the electron renderer process at:

  ➜  Local:   http://127.0.0.1:5173/
  ➜  Network: use --host to expose
error during start dev server and electron app:
Error: No electron app entry file found: /Users/Rappa/Desktop/NativePHP/vendor/nativephp/electron/resources/js/out/main/index.js
    at ensureElectronEntryFile (/Users/Rappa/Desktop/NativePHP/vendor/nativephp/electron/resources/js/node_modules/electron-vite/dist/chunks/lib-3592fc5a.js:26:23)
    at Object.startElectron (/Users/Rappa/Desktop/NativePHP/vendor/nativephp/electron/resources/js/node_modules/electron-vite/dist/chunks/lib-3592fc5a.js:120:5)
    at createServer (/Users/Rappa/Desktop/NativePHP/vendor/nativephp/electron/resources/js/node_modules/electron-vite/dist/chunks/lib-86ea98bb.js:74:21)
    at async CAC.<anonymous> (/Users/Rappa/Desktop/NativePHP/vendor/nativephp/electron/resources/js/node_modules/electron-vite/dist/cli.js:48:9)
error Command failed with exit code 1.
info Visit https://yarnpkg.com/en/docs/cli/run for documentation about this command.
```

### How to reproduce the bug

```
laravel new NativePHP
composer require nativephp/electron
php artisan native:install
php artisan native:serve <-- error
```

### Package Version

0.1.0

### PHP Version

8.2

### Laravel Version

10.0

### Which operating systems does with happen with?

macOS

**Comment by daltonsutton** [comment]
Getting the same error.

**Comment by jonsugar** [comment]
Same as #10

**Comment by mpociot** [comment]
Fixed in 0.1.1
